airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/StagingFilenameGenerator.java

```diff
@@ -21,9 +22,14 @@
   // file suffix; its value range will be [1, maxPartsPerFile]
   private int currentFileSuffixPartCount = 0;
 
-  public StagingFilenameGenerator(final String streamName, final int maxPartsPerFile) {
+  // This variable is responsible to set the size of chunks size (In MB). After chunks created in
+  // S3 or GCS they will be uploaded to Snowflake or Redshift. These service have some limitations for the uploading file.
+  // So we make the calculation to determine how many parts we can put to the single chunk file.
+  private final long chunkLimit;
+
+  public StagingFilenameGenerator(final String streamName, final long chunkLimit) {
```

Comment:
nitpick: rename the argument from `chunkLimit` --> `chunkSize` to clarify that it represents the size (in bytes). also a javadoc comment would be nice

Reply:
Make sense. This was done